src/System/Taffybar/SimpleConfig.hs

```diff
@@ -135,7 +136,10 @@ simpleTaffybar :: SimpleTaffyConfig -> IO ()
 simpleTaffybar conf = dyreTaffybar $ toTaffyConfig conf
 
 getMonitorCount :: IO Int
-getMonitorCount = screenGetDefault >>= maybe (return 0) screenGetNMonitors
+getMonitorCount = do
```

Comment:
Why was this changed so much? Just to add the fromIntegral? Why not just add a fromIntegral <$> out front?